package/features/environment.py

```diff
@@ -26,6 +30,29 @@ def before_scenario(context, scenario):
     Installs kedro by running pip in the top level directory.
     """
 
+    kedro_version = Version.parse("1.0.0")
```

Comment:
I'm confused, shouldn't this be 

```
import kedro

kedro_version = Version.parse(kedro.__version__)
```

?

Reply:
kedro_version was just a placeholder but yes, can replace it with kedro.__version__ 

I will make the semver change to packaging in another PR. As we will need to update it in multiple places.

Reply:
Thanks for the clarification! Maybe adding a code comment would make it more clear